### DISCUSSION TITLE
New private marketplace feature

### DISCUSSION THREAD

**Discussion by lucascanna** [mia-platform/community#282] [Q&A] [answered]
### Which section?

Console

### Which is the version?

11.3.0

### Description

Hi!

Can you confirm that with the new feature of the "private marketplace", a user outside of Mia-Platform can access to the CMS of the PaaS console and autonomously add its template? 

Does he need some permissions in order to access the CMS?

Thanks in advance!

**Comment by fredmaggiowski** [comment]
Hi, no external user cannot access the CMS on our SaaS

**Comment by davidebianchi** [comment]
We are working to manage the marketplace with [miactl](https://github.com/mia-platform/miactl), our cli. We should release it in the v12 of the platform.

**Reply by akelity** [reply, marked as answer]
Here the documentation on how to add an item to the marketplace here: https://docs.mia-platform.eu/docs/marketplace/add_to_marketplace/contributing_overview
